tests/test_pubmed_oa_parser.py

```diff
@@ -39,7 +39,6 @@ def test_parse_pubmed_xml():
     assert parsed_xml.get("doi") == "10.1371/journal.pone.0046493"
     assert parsed_xml.get("subjects") == "Research Article; Biology; Biochemistry; Enzymes; Enzyme Metabolism; Lipids; Fatty Acids; Glycerides; Lipid Metabolism; Neutral Lipids; Metabolism; Lipid Metabolism; Proteins; Globular Proteins; Protein Classes; Recombinant Proteins; Biotechnology; Microbiology; Bacterial Pathogens; Bacteriology; Emerging Infectious Diseases; Host-Pathogen Interaction; Microbial Growth and Development; Microbial Metabolism; Microbial Pathogens; Microbial Physiology; Proteomics; Sequence Analysis; Spectrometric Identification of Proteins"  # noqa
     assert "Competing Interests: " in parsed_xml.get("coi_statement")
-    assert parsed_xml.get('publication_date') == '28-9-2012'
```

Comment:
This change doesn't belong here

Reply:
I did a [commit](https://github.com/titipata/pubmed_parser/pull/142/commits/0bd12bff5a7458b27c9f5a445a1905d95ad3694b) to revert the changes. You shouldn't be able to see them in [files changed](https://github.com/titipata/pubmed_parser/pull/142/files). If you want the changes (and the undo) to not show up in the commit history, I can do a new PR.

Reply:
Yes you're right, it doesn't show up anymore